tests/test_assets.py

```diff
@@ -315,7 +321,8 @@ def test_to_and_from_dict(self):
         self.assertTrue('notice_date' in dictd)
         self.assertTrue('expiration_date' in dictd)
         self.assertTrue('auto_close_date' in dictd)
-        self.assertTrue('contract_multiplier' in dictd)
+        self.assertTrue('tick_size' in dictd)
```

Comment:
Unrelated to this change, but why does this not assert anything about the contents of the dict here?  It would be the same amount of work for this to do:

```
self.assertEqual(dictd['root_symbol'], <expected symbol>)
...
```
or better yet:

```
self.assertEqual(self.future.to_dict(), {key: getattr(self.future, key) for key in dir(Future))
```